src/EditorFeatures/TestUtilities2/Intellisense/TestStateFactory.vb

```diff
@@ -28,10 +27,8 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
             testState.Workspace.SetOptions(
                 testState.Workspace.Options.WithChangedOption(CompletionOptions.TriggerInArgumentLists, LanguageNames.CSharp, showCompletionInArgumentLists))
 
-            If ensureCompeltionProvidersAvailable Then
-                Dim completionService = DirectCast(testState.Workspace.Services.GetLanguageServices(LanguageNames.CSharp).GetRequiredService(Of CompletionService)(), CompletionServiceWithProviders)
-                completionService.GetTestAccessor().EnensureCompeltionProvidersAvailable()
-            End If
+            Dim completionService = DirectCast(testState.Workspace.Services.GetLanguageServices(LanguageNames.CSharp).GetRequiredService(Of CompletionService)(), CompletionServiceWithProviders)
+            completionService.GetTestAccessor().EnsureCompletionProvidersAvailable()
```

Comment:
i woudl not have tests have separate behavior.  i think the CompServiceWithProviders should always have these availab.e.